recipes/pycairo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.10.0" %}
+{% set checksum = "9bca41bf9974cff78b9c9b677feacd0c413e79b16cf31ab5d3a3de4e9c48d6db" %}
+{% set checktype = "sha256" %}
+
+package:
+    name: pycairo
+    version: {{ version }}
+
+source:
+    fn: master.zip
+    url: https://github.com/Jactry/pycairo/archive/master.zip
```

Comment:
Ouch, that is pretty ancient. Normally in these situations we try to get upstream to cut a fresh release for us. Have opened this issue ( https://github.com/Jactry/pycairo/issues/1 ) to see if we can make this happen.